eng/Directory.Build.Data.props

```diff
@@ -112,7 +112,6 @@
     <DelaySign>false</DelaySign>
     <PublicSign>false</PublicSign>
     <ImportDefaultReferences>false</ImportDefaultReferences>
-    <UseProjectReferenceToAzureCore>true</UseProjectReferenceToAzureCore>
```

Comment:
Do we want to totally remove this switch as opposed to just setting it to false? Does it not still hold some value possibly for testing?

Reply:
cc @weshaggard

Reply:
I would rather we add it or something similar back when the testing story gets figured out.

Reply:
Because it will have support other libraries in addition to Azure.Core like Azure.Identity and Azure.S.Blob